src/ci/docker/host-x86_64/mingw-check-tidy/Dockerfile

```diff
@@ -35,4 +35,4 @@ COPY host-x86_64/mingw-check/validate-error-codes.sh /scripts/
 # NOTE: intentionally uses python2 for x.py so we can test it still works.
 # validate-toolstate only runs in our CI, so it's ok for it to only support python3.
 ENV SCRIPT TIDY_PRINT_DIFF=1 python2.7 ../x.py test \
-           --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint,cpp:fmt
+           --stage 0 src/tools/tidy tidyselftest --extra-checks=py,cpp
```

Comment:
if python formatting fails, does the error message tell you how to format the files with tidy yourself? if no that would be useful, since having to pass `--extra-checks=py --bless` is a bit obscure

Reply:
Good point!

Reply:
https://github.com/rust-lang/rust/pull/134968